pkg/pki/ca/ca.go

```diff
@@ -22,10 +22,29 @@ import (
 	"errors"
 	"time"
 
+	"github.com/golang/glog"
 	"istio.io/auth/pkg/pki"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	apiv1 "k8s.io/client-go/pkg/api/v1"
 )
 
 const (
+	// The Istio secret annotation type
+	IstioSecretType = "istio.io/key-and-cert"
+
+	// The ID/name for the certificate chain file.
+	CACertChainID = "ca-cert.pem"
+	// The ID/name for the private key file.
+	CAPrivateKeyID = "ca-key.pem"
+	// The ID/name for the CA root certificate file.
+	RootCertID = "root-cert.pem"
+
+	CAServiceAccount = "istio-ca-creds"
```

Comment:
We can't assume the service account is a const, can we?

Reply:
It's actually not a service account, it's the secret name. Do we actually need this annotation?